lib/index.js

```diff
@@ -191,12 +191,16 @@ CloudFront.prototype.request = function(method, path, headers, cb) {
 
       cb(err, info);
     });
-    parser.on("error", function(err) {
-      cb(err, info);
+    parser.on('error', function(err) {
+      cb(err);
```

Comment:
Oops, sorry no reason. I actually follow the same signature for the error on req. Putting it back.